src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -71,6 +73,22 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
+        public static void Close() {
+            var folder = GetTempCsvFilesFolder();
+            if (Directory.Exists(folder)) {
+                foreach (var f in Directory.EnumerateFiles(folder)) {
+                    try {
+                        File.Delete(f);
+                    } catch(IOException) { } catch (UnauthorizedAccessException) { }
+                }
+            }
+        }
+
+        private static string GetTempCsvFilesFolder() {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
(And if they use "Open", and then decide that they want to save it later, the proper workflow is to do "Save As" in Excel, and place it wherever they want it to be, and give it a name that they will remember.)